common/utils.js

```diff
@@ -0,0 +1,47 @@
+export function findById(items, id) {
```

Comment:
this file looks pretty close to a copy/paste job, including the comments. Please try to use class code as a resource to code against rather than a place to pull code from